feat: normal spellbook sounds

DesecratedTree opened the pull request:
## What has been done?
added the sounds for all combat spells minus surge spells (cannot find sound ids anywhere)
added the sounds for alching
added sounds for enchanting
fixed the telegrab sound. 3008 is the sound for vulnerability
fixed some enchantment item ids (some were blank or placeholder items)

## Has your code been documented?

alycii commented on game/plugins/src/main/kotlin/gg/rsmod/plugins/content/combat/strategy/MagicCombatStrategy.kt:
```diff
@@ -64,6 +64,28 @@ object MagicCombatStrategy : CombatStrategy {
         if (pawn is Player) {
             MagicSpells.getMetadata(spell.uniqueId)
                 ?.let { requirement -> MagicSpells.removeRunes(pawn, requirement.runes) }
+            /* ------- SPELL SOUNDS -------- */
```
It's better to put the sounds into the spells enum, rather than having multiple lines like this.

`pawn.playSound(spell.sound)`

DesecratedTree replied:
resolved

DesecratedTree commented:
- switched sounds to enum
- changed sound for picking up items with telegrab
- all enchanting sound values will return null since those are different per product and are handled elsewhere
- ^^ same with teleports
- alchemy, superheat sounds
- sounds for most normal spellbook spells
- currently surge spells are missing sounds, but I am using the sounds from the previous strength level of spells
